change/@uifabricshared-theming-react-native-2020-07-02-16-00-55-web-tester.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I'm surprised that it generated changes for packages you didn't touch. Might be a bad carry-over from something else.